lnrpc/peersrpc/peers.proto

```diff
@@ -1,10 +1,91 @@
 syntax = "proto3";
 
+import "lightning.proto";
+
 package peersrpc;
 
 option go_package = "github.com/lightningnetwork/lnd/lnrpc/peersrpc";
 
 // Peers is a service that can be used to get information and interact
 // with the other nodes of the newtwork.
 service Peers {
+    /*
+    UpdateNodeAnnouncement allows the caller to update the node parameters
+    and broadcasts a new version of the node announcement to its peers.
+    */
+    rpc UpdateNodeAnnouncement (NodeAnnouncementUpdateRequest)
+        returns (NodeAnnouncementUpdateResponse);
+}
+
+// UpdateAction is used to determine the kind of action we are referring to.
+enum UpdateAction {
+    ADD = 0;
+    REMOVE = 1;
```

Comment:
Looks like something was overwritten somewhere, but the API doc comment here is still missing.

Reply:
I think I got it wrong. Added the comment for `UpdateAction` and the changes are in `lnrpc/peersrpc/peers.proto` but I guess you were asking to add comments for the `ADD` and `REMOVE` options.

Reply:
Ah, yes. Sorry, I didn't even notice you added the comment to `UpdateAction`. The gRPC API doc generator is weird in that way, AFAIK you cannot really document the enum itself, only its elements.